DOCKER.md

```diff
@@ -0,0 +1,63 @@
+# Cross compiling with `Docker`
+The manual process kind of pollutes your workstation by setting lots of environment variables and writing to various config files. Thus it can be difficult to remove or upgrade these files and settings or even repeat that process for different versions of rust on the same machine.
```

Comment:
It's alright. That's why it's nice to have someone reviewing the changes :smile: Both methods are good for different reasons, as you wrote in the readme, which is why I prefer to keep things somewhat neutral.